src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -189,6 +190,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return v;
     }
 
+    public ProgressBar getProgressBar(){
```

Comment:
If OCFileListFragment extends ExtendedListFragment, you don't need to create this method getProgressBar(), you could just use the one created in ExtendedListFragment.